src/tools/rustfmt/config_proc_macro/src/attrs.rs

```diff
@@ -68,7 +68,11 @@ fn get_name_value_str_lit(attr: &syn::Attribute, name: &str) -> Option<String> {
     match &attr.meta {
         syn::Meta::NameValue(syn::MetaNameValue {
             path,
-            value: syn::Expr::Lit(syn::ExprLit { lit: syn::Lit::Str(lit_str), .. }),
+            value:
+                syn::Expr::Lit(syn::ExprLit {
+                    lit: syn::Lit::Str(lit_str),
+                    ..
+                }),
```

Comment:
I think it would be better to make this formatting change by opening a PR in https://github.com/rust-lang/rustfmt. When you get a chance please revert this change.

Reply:
Done, thanks 👍